lib/pf/factory/condition/profile.pm

```diff
@@ -59,6 +59,18 @@ sub instantiate {
             my $cond_and = pf::condition::all->new(conditions => [$cond1, $cond2]);
             return $cond_and;
         }
+        elsif($type eq 'exists_in'){
+            my @values = split(/ *; */, $data->{value});
```

Comment:
My bad I saw it's done.
But shouldn't we use \s like in the other parts of the code ?